src/main/java/org/dita/dost/ant/ExtensibleAntInvoker.java

```diff
@@ -261,14 +264,12 @@ private AbstractPipelineModule getPipelineModule(final ModuleElem m, final Pipel
                 }
             }
             final AbstractPipelineModule module = factory.createModule(m.getImplementation());
-            try {
-                module.setProcessingPipe(m.getFilters());
-            } catch (IllegalAccessException | InstantiationException e) {
-                throw new BuildException(e);
-            }
+            module.setParallel(m.parallel);
+            module.setProcessingPipe(m.getFilters());
             if (!m.fileInfoFilters.isEmpty()) {
                 module.setFileInfoFilter(combine(m.fileInfoFilters));
             }
+            module.setParallel(m.parallel);
```

Comment:
Curious - why do we need to run `module.setParallel(m.parallel);` on lines 267 and 272? Does the FileInfoFilter reset it?

Reply:
Good catch, duplicate setter call.